Sample/Vungle Sample App/BannerList.m

```diff
@@ -0,0 +1,139 @@
+    if (indexPath.row == (self.totalSize - 4)) {
+        height = 250.0;
+    }
+    return height;
+}
+-(UITableViewCell *)tableView:(UITableView*) tableView cellForRowAtIndexPath:(nonnull NSIndexPath *)indexPath {
+    NSError *error = nil;
+    UITableViewCell *cell;
+        if ((indexPath.row == 0) || (indexPath.row == (self.totalSize / 2)) || (indexPath.row == 1)) {
+           
+            cell = [self.tableView dequeueReusableCellWithIdentifier:@"adCell" forIndexPath:indexPath];
+                    CGFloat cellWidth = cell.contentView.frame.size.width;
+            DisplayTableViewCell *bannerCell = (DisplayTableViewCell*) cell;
+            bannerCell.adView.frame = CGRectMake((cellWidth - 320)/2.0, 10, 320.0, 50.0);
+            [self displayBannerAd:kVungleTestPlacementID07];
+            [self.sdk addAdViewToView:bannerCell.adView withOptions:@{} placementID:kVungleTestPlacementID07 error:&error];
+             NSLog(@"The ID is: %@",kVungleTestPlacementID07);
+                   
+                   
+        }
+        else if (indexPath.row == (self.totalSize - 4)) {
+            cell = [self.tableView dequeueReusableCellWithIdentifier:@"mrecCell" forIndexPath:indexPath];
+                    CGFloat cellWidth = cell.contentView.frame.size.width;
+            DisplayTableViewCell *bannerCell = (DisplayTableViewCell*) cell;
+            
+            bannerCell.mrecView.frame = CGRectMake((cellWidth - 320)/2.0, 10, 300.0, 250.0);
+            if (![self.sdk loadPlacementWithID:kVungleTestPlacementID05 error:&error]) {
+                NSLog(@"Failed to load MREC");
+            }
+            [self.sdk addAdViewToView:bannerCell.mrecView withOptions:nil placementID:kVungleTestPlacementID05 error:&error];
+            NSLog(@"The ID is %s",kVungleTestPlacementID05);
+            
+        }
+        else {
+            NSLog(@"It will load a picture ad");
+            cell = [self.tableView dequeueReusableCellWithIdentifier:@"displayCell" forIndexPath:indexPath];
+            cell.imageView.image = [UIImage imageNamed:@"vungle_logo_aqua_mango_banner"];
+        }
+    return cell;
+}
+- (NSInteger)tableView:(UITableView *)tableView numberOfRowsInSection:(NSInteger)section{
+    return self.totalSize;
+}
+
+- (void)startVungle {
+    NSLog(@"Starting the VungleSDK in banner");
+    self.sdk = [VungleSDK sharedSDK];
+    [self.sdk setDelegate:self];
+    [self.sdk setLoggingEnabled:YES];
+    NSError *error = nil;
+    if(![self.sdk startWithAppId:kVungleTestAppID error:&error]) {
+        NSLog(@"Error while starting VungleSDK %@", [error localizedDescription]);
+        return;
+    }
+    NSLog(@"Attempting to load banner ad in startVungle");
+}
+
+-(void) displayBannerAd: (NSString *)placementId {
+    NSError* error = nil;
+    NSLog(@"Attempting to load banner ad");
+    if ([self.sdk loadPlacementWithID:placementId withSize:VungleAdSizeBanner error:&error]) {
+        NSLog(@"Playing the banner ad");
```

Comment:
If you're including the NSError object in the method call, you might want to include the `localizedDescription` in your NSLog output. This might be beneficial to end users.

Reply:
Obviously, this would need to be captured in an `else` condition in this case.